zipkin-lens/src/translations/es/messages.json

```diff
@@ -14,6 +15,7 @@
   "Filter": "Filtro",
   "Limit": "Límite",
   "None": "Ninguno",
+  "One Result": "",
```

Comment:
@jeqo @jcchavezs can you help with this and also "{0} Results"?

Reply:
```suggestion
  "One Result": "Un Resultado",
```